src/gui/aboutdialog.cpp

```diff
@@ -36,9 +36,12 @@
 #include "uithememanager.h"
 #include "utils.h"
 
+#define SETTINGS_KEY(name) "AboutDialog/" name
+
 AboutDialog::AboutDialog(QWidget *parent)
     : QDialog(parent)
     , m_ui(new Ui::AboutDialog)
+    , m_storeDialogSize(SETTINGS_KEY("Dimension"))
```

Comment:
Why not "Size"?
"Dimension" isn't correct name for this setting. You store dialog "size". What could have made you use another term to denote it?

Reply:
>"Dimension" isn't correct name for this setting. You store dialog "size". What could have made you use another term to denote it?

We already use it in a few dialogs, it is not new.
Do you suggest we change all of them?

>Why not "Size"?

"Size" itself is vague when we are referring to the "dialog size".

Reply:
>We already use it in a few dialogs, it is not new.
Do you suggest we change all of them?

At least don't use it in new ones.
>"Size" itself is vague when we are referring to the "dialog size" IMO.

When I see something like
```ini
[AboutDialog]
Size=
```
it is clear enough for me. But you can be more details, e.g. "WindowSize".
The term Size is most commonly used in this sense. What exactly is wrong with it?

Reply:
>But you can be more details, e.g. "WindowSize".

How about "DialogSize"? I would like to unify the instances.

>What exactly is wrong with it?

I already told my viewpoint and I don't really resist changing to it.
"Dimension" isn't out of context: https://dictionary.cambridge.org/us/dictionary/english/dimension

Reply:
>"Dimension" isn't out of context

It isn't out of context. They are related, of course. But...

>Dimension is **a measurement of something in a particular direction**, especially its height, length, or width

But size that are stored by you consists of two dimensions.

But the main thing is that it raises the question, why is an unusual term used to refer to such an ordinary thing as size? I hate subjective arguments, but is the question "why is the size called not the size" subjective?
Generally, I expressed what it causes. Let you evaluate it yourself and make the final decision.

Reply:
>I hate subjective arguments, but is the question "why is the size called not the size" subjective?

Then you question found the wrong person, I'm not the first to use it.

>Let you evaluate it yourself and make the final decision.

If no one objects then I'll settle on "DialogSize".

Reply:
>Then you question found the wrong person, I'm not the first to use it.

But you still found it appropriate to follow it...
>If no one objects then I'll settle on "DialogSize".

Too verbose, of course... but it's better.

Reply:
I vote for ```Size```